include/bluetooth/mcc.h

```diff
@@ -500,12 +530,36 @@ int bt_mcc_read_segments_obj_id(struct bt_conn *conn);
 /** @brief Read Current Track Object ID */
 int bt_mcc_read_current_track_obj_id(struct bt_conn *conn);
 
+/** @brief Set Current Track Object ID
+ *
+ * Set the Current Track to the the track given by the @p id parameter
+ *
+ * @param id   Object Transfer Service ID (UINT48) of the track to set as the current track
+ */
+int bt_mcc_set_current_track_obj_id(struct bt_conn *conn, uint64_t id);
+
 /** @brief Read Next Track Object ID */
 int bt_mcc_read_next_track_obj_id(struct bt_conn *conn);
 
+/** @brief Set Next Track Object ID
+ *
+ * Set the Next Track to the the track given by the @p id parameter
+ *
+ * @param id   Object Transfer Service ID (UINT48) of the track to set as the next track
+ */
+int bt_mcc_set_next_track_obj_id(struct bt_conn *conn, uint64_t obj_id);
+
 /** @brief Read Current Group Object ID */
 int bt_mcc_read_current_group_obj_id(struct bt_conn *conn);
 
+/** @brief Set Current Group Object ID
+ *
+ * Set the Current Group to the the group given by the @p id parameter
+ *
+ * @param obj_id   Object Transfer Service ID (UINT48) of the group to set as the current group
+ */
+int bt_mcc_set_current_group_obj_id(struct bt_conn *conn, uint64_t obj_id);
+
```

Comment:
This is kinda fun:
In `bt_mcc_set_current_track_obj_id` you reference `@p id`, document `id` and the argument is `id` - All is good
In `bt_mcc_set_next_track_obj_id` you reference `@p id`, document `id` but the argument is `obj_id` - One issue here
In `bt_mcc_set_current_group_obj_id` you refernce `@p id`, document `obj_id` and the argument is `obj_id` - One or two issue here. 

A few consistency fixes are needed here

Reply:
I thought I had changed them all, but obviously a few slipped in the hurry yesterday.
Fixed now.